server.js

```diff
@@ -1,9 +1,43 @@
 const express = require('express');
+
+
+const { startDatabase, stopDatabase, isConnected } = require('./db');
+const app = express();
+
+
+app.get('/', (req, res) => {
+  res.json({
+    message: 'o_O',
+    database: isConnected() ? 'connected' : 'disconnected'
+  })
+});
+
+
+process.on('SIGINT', async () => {
+  await stopDatabase();
+  process.exit(0);
+});
+
+process.on('SIGTERM', async () => {
+  await stopDatabase();
+  process.exit(0);
+});
+
+if (require.main === module) {
+  app.listen(8000, async () => {
+    await startDatabase();
+
+    console.log(`🚀 server running on PORT: $`);
```

Comment:
The console log message in the `app.listen` callback is missing the port number. You should include the port number in the message for better debugging. [medium]